src/view/use-announcer/use-announcer.js

```diff
@@ -42,9 +42,11 @@ export default function useAnnouncer(contextId: ContextId): Announce {
         // unmounting after a timeout to let any announcements
         // during a mount be published
         setTimeout(function remove() {
-          // not clearing the ref as it might have been set by a new effect
-          getBodyElement().removeChild(el);
-
+          // checking if element exists before remove to prevent errors
+          if (getBodyElement().contains(el)) {
```

Comment:
I just checked - this is all good for ie11

https://developer.mozilla.org/en-US/docs/Web/API/Node/contains